common/environment/environment.cpp

```diff
@@ -122,6 +121,11 @@ class CLocalEnvironment : public CInterface, implements IConstEnvironment
     void buildMachineCache() const;
     void buildDropZoneCache() const;
     void init();
+    void createComputerNameXpath(const char * computer, StringBuffer &xpath)  const
+    {
+        xpath.set("numberOfDropZones[@name=\"");
+        xpath.append(computer).append("\"]/number");
+    }
```

Comment:
style/minor: we tend have functions that fill StringBuffers return the StringBuffer too (like the append methods of StringBuffer), so that you can chain them.

e.g. 
getComputerNameXpath(<computer>, result).append(<somethingelse>).

In this case you'll not use the return, but it's more flexible in general.